ts/sagas/wallet/cobadgeReminder.ts

```diff
@@ -0,0 +1,96 @@
+import { IndexedById } from "../../store/helpers/indexer";
+import { StatusEnum } from "../../../definitions/pagopa/cobadge/configuration/CoBadgeService";
+import { NetworkError } from "../../utils/errors";
+import { loadCoBadgeAbiConfiguration } from "../../features/wallet/onboarding/cobadge/store/actions";
+import { sendAddCobadgeMessage } from "../../store/actions/wallet/wallets";
+
+/**
+ * This saga aims to send an event to Mixpanel to get information about whether the user has a bancomat card with which
+ * it is allowed to add a co-badge card and has not yet done or not.
+ *
+ * This saga is called only if the {@link bancomatListVisibleInWalletSelector} return some
+ */
+export function* sendAddCobadgeMessageSaga() {
+  // Check if there is at least one bancomat
+  const maybeBancomatListVisibleInWallet: pot.Pot<
+    ReadonlyArray<BancomatPaymentMethod>,
+    Error
+  > = yield select(bancomatListVisibleInWalletSelector);
+
+  const bancomatListVisibleInWallet = pot.getOrElse(
+    maybeBancomatListVisibleInWallet,
+    []
+  );
+
+  if (bancomatListVisibleInWallet.length === 0) {
+    yield put(sendAddCobadgeMessage(false));
+    return;
+  }
+
+  // Extract the cobadgeAbi if there is at least a cobadge card
+  const maybeCobadgeVisibleInWallet: pot.Pot<
+    ReadonlyArray<CreditCardPaymentMethod>,
+    Error
+  > = yield select(cobadgeListVisibleInWalletSelector);
+
+  const cobadgeVisibleInWallet = pot.getOrElse(maybeCobadgeVisibleInWallet, []);
+  const cobadgeAbis = cobadgeVisibleInWallet
+    .filter(c => c.info.issuerAbiCode !== undefined)
+    .map(cWithAbi => cWithAbi.info.issuerAbiCode);
+
+  // Check if the abiConfiguration is Some
+  // and if not request the abiConfiguration
+  if (!pot.isSome(yield select(coBadgeAbiConfigurationSelector))) {
+    yield put(loadCoBadgeAbiConfiguration.request());
+
+    // Wait for the request results
+    const loadCoBadgeAbiRes = yield take([
+      getType(loadCoBadgeAbiConfiguration.success),
+      getType(loadCoBadgeAbiConfiguration.failure)
+    ]);
+
+    // If the request result is failure return
+    if (isActionOf(loadCoBadgeAbiConfiguration.failure, loadCoBadgeAbiRes)) {
+      return;
+    }
+  }
+  const maybeCoBadgeAbiConfiguration: pot.Pot<
+    IndexedById<StatusEnum>,
+    NetworkError
+  > = yield select(coBadgeAbiConfigurationSelector);
+
+  if (pot.isSome(maybeCoBadgeAbiConfiguration)) {
+    const coBadgeAbiConfiguration = maybeCoBadgeAbiConfiguration.value;
+
+    // Extract a list of abi that satisfy the following conditions:
+    // - is a bancomat in the wallet of the user
+    // - the abi of the bancomat is in the abiConfiguration list
+    // - the abi of the bancomant has the status enabled in the abiConfiguration list
+    // - there isn't a cobadge card in the wallet of the user with the sami abi
+    const enalbedAbis = bancomatListVisibleInWallet.filter(
```

Comment:
```suggestion
    const enabledAbis = bancomatListVisibleInWallet.filter(
```

Reply:
Fixed in [715c878](https://github.com/pagopa/io-app/pull/2952/commits/715c87864b660a03adb108b5e9b7446349606f6e)